src/App.tsx

```diff
@@ -14,7 +14,9 @@ function App() {
           <Navbar />
           <Routes>
             <Route path='/' element={<HomePage />} />
-            <Route path='/users/:id' element={<UserDetailPage />} />
+            <Route path='/users/details' element={<UserDetailPage />} />
+            {/* path='/users/:id' */}
+            {/* потом будет заимплементено */}
```

Comment:
We usually write `TODO: and some shore descriotion of what you need to do in the future`

Reply:
Also, always use English, even in your learning PRs